frontend/online-shop/src/App.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./App.css";
+import ProductList from "./components/ProductList";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Header from "./components/Header";
+import ProductDetails from "./components/ProductDetails";
+
+function App() {
+  return (
+    <Router>
+      <Header />
+      <Switch>
+        <Route path="/products/:productId" component={ProductDetails} />
+        <Route path="/products">
+          <ProductList />
+        </Route>
+        <Route path="/">
+          <ProductList />
```

Comment:
Instead of having two routes that render `ProductList` you could ass a `Redirect` that will redirect the `/` path towards the `products` path that will render the `ProductList`